.claude/ralph-loop.local.md

```
@@ -0,0 +1,23 @@
+
+Fix bug: [DESCRIPTION]
+
+  Steps:
+  1. Run 
+  2. Identify root cause(s)
+  3. Implement fix(s)
+  4. Verify fix works
+  5. Check no new issues introduced
+
+  After 15 iterations if not fixed:
+  - Document blocking issues
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)